scripts/scil_score_tractogram.py

```diff
@@ -0,0 +1,359 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+"""
+Scores input tractogram overall and bundlewise. Outputs a results.json
+containing a full report and splits the input tractogram into resulting
+tractogram : *_tc.tck, *_fc.tck, nc.tck and *_wpc.tck,
+where * is the current bundle.
+
+Definitions:
+    tc: true connections, streamlines joining a correct combination
+        of ROIs.
+    fc: false connections, streamlines joining an incorrect combination of
+        ROIs.
+    nc: no connections, streamlines not joining two ROIs.
+    wpc: wrong path connections, streamlines that go outside of the ground
+        truth mask, joining a correct combination of ROIs.
+    Bundle overlap : ground truth voxels containing tc streamline(s).
+
+Either input gt_endpoints or gt_heads and gt_tails. Ground truth and ROIs
+must be in the same order i.e. groundTruth1.nii.gz .... groundTruthN.nii.gz \
+    --gt_tails tail1.nii.gz ... tailN.nii.gz \
+    --gt_heads head1.nii.gz ... headN.nii.gz
+
+Masks can be dilated with --dilate_endpoints for bundle recognition.
+
```

Comment:
Can you provide a sample command line example please. A bit like in your PR conversation. I know it is a bit long but because the script support "complicated" input options, your example is great and will help users in the future.

Reply:
Otherwise, super clear help and description! Good job @AntoineTheb

Reply:
Thanks ! I have updated the doc to include an example in 51feb57